Project.toml

```diff
@@ -18,7 +18,7 @@ ShaderAbstractions = "65257c39-d410-5151-9873-9b3e5be5013e"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
 [compat]
-AbstractPlotting = "0.15.4"
+AbstractPlotting = "0.15"
```

Comment:
This actually needs to be 0.15.4

Reply:
Okay!
Any other change?

Reply:
Still works!!!

Reply:
@SimonDanisch Its really odd that I just changed Julia versions and it works...🙂
Any idea, as to why thats the case?